pkg/filters/util.go

```diff
@@ -52,3 +56,8 @@ func buildSortingRequest(c Filters) *admin.Sort {
 	}
 	return nil
 }
+
+func getToken(c Filters) string {
+	token := int(c.Page-1) * int(c.Limit)
```

Comment:
Why do we need this ?

Reply:
I think the `token` is actually the offset i.e. the number of items to skip while fetching. In the issue, we wanted to build a page-based flag. So, this actually calculates the number of items to skip using `page` and `limit`.

Reply:
Just verified that it's the offset. https://github.com/flyteorg/flyteadmin/blob/60110119d8919403c43f74c781d59a380f3d1f21/pkg/manager/impl/task_manager.go#L166

Reply:
correct